lib/src/http/auto_retry.dart

```diff
@@ -129,6 +129,12 @@ class RequestAutoRetry extends AutoRetryConfig {
   /// This will be incremented each time an attempt is sent.
   int numAttempts = 0;
 
+  /// This number is used in the advanced jitter algorithm, and has no
+  /// meaningful value outside of that context.
+  ///
+  /// Do not modify this value.
+  double previous = 0.0;
```

Comment:
+1, if possible.

Reply:
@evanweible-wf @maxwellpeterson-wf Took a stab at this [here](https://github.com/Workiva/w_transport/pull/337/commits/fe0ba4ede3d9ab1cc3a723fc526fcd2806d36bd9)

It's a little convoluted since the utils and the request object are in different packages (so the utils package can't access any internal vars on the request). LMK what you think.

Reply:
That approach seems fine to me, can we remove this field now?

Reply:
Derp, yea I just forgot to remove it. Nice catch

Reply:
Removed